projects/questionset-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -153,6 +154,10 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
         this.getFrameworkDetails(this.primaryCategoryDef);
       }
     });
+    this.editorService.readComment(this.collectionId).subscribe((res) => {
```

Comment:
@ramkumar-pacewisdom This will be called everytime we load the editor, can it be called on on-demand basis?

Reply:
i tried when user click on Comments Pop-up, But in slow internet devices the response is slow and draft data not rendering on time to the user.